Marlin/Marlin_main.cpp

```diff
@@ -11503,6 +11768,27 @@ void setup() {
   #if ENABLED(ENDSTOP_INTERRUPTS_FEATURE)
     setup_endstop_interrupts();
   #endif
+
+  //Turn on White at Powerup
+  #if ENABLED(RGB_STRIP) && ENABLED(LIGHT_ON_POWERUP) 
+    digitalWrite(RGB_STRIP_R_PIN, HIGH);
+    digitalWrite(RGB_STRIP_G_PIN, HIGH);
+    digitalWrite(RGB_STRIP_B_PIN, HIGH);
+    analogWrite(RGB_STRIP_R_PIN, 255);
+    analogWrite(RGB_STRIP_G_PIN, 255);
+    analogWrite(RGB_STRIP_B_PIN, 255);
+  #elif ENABLED(RGBW_STRIP) && ENABLED(LIGHT_ON_POWERUP)
+    digitalWrite(RGB_STRIP_W_PIN, HIGH);
+    analogWrite(RGB_STRIP_W_PIN, 255);
+  #endif
```

Comment:
It would be preferable to use the relevant `set_led_color` function here. The code will be more maintainable.